crates/rolldown_ecmascript_utils/src/extensions/ast_ext/expression_ext.rs

```diff
@@ -79,4 +81,15 @@ impl<'ast> ExpressionExt<'ast> for ast::Expression<'ast> {
     matches!(self, ast::Expression::StaticMemberExpression(member_expr)
     if member_expr.object.is_import_meta() && member_expr.property.name == "url")
   }
+
+  /// Check if the expression is `import.meta.hot.accept`
```

Comment:
The comments seems put in the wrong place.

Reply:
Solved.